CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 ## Changelog
 
+### 0.16.0 [2022-01-20]
+
+#### General
+
+* Added support for OpenSSL cryptography backend so RNP may be built and used on systems without the Botan installation.
```

Comment:
maybe "without Botan installed" ?

Reply:
Ok, fixing! Any other suggestions (to not force-push after the every change).

Reply:
no, looks good

Reply:
Thanks, re-pushed!